source/ui/Image.js

```diff
@@ -173,14 +180,15 @@
 		*/
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
+			var multiple= "url('"+ enyo.path.rewrite(src) +"'), url('"+ this.placeholder + "');";
```

Comment:
I will do the required changes to check for falsy value of `placeholder`. And if no placeholder defined, then default placeholder to be used. 
About proper support of placeholder image assets, Does someone from Enyo SVL will look into `rewrite` aspect or I have to come up with something to support.